raven-os/corefs/0.1.0/manifest.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""
+Build manifest for the 'corefs' package.
+The corefs package contains the basic filesystem of Raven-OS.
+"""
+
+from textwrap import dedent
+from nbuild.stdenv.package import package
+from nbuild.stdenv.install import make_keeper, make_symlink
+
+
+@package(
+    id=f'stable::raven-os/corefs#0.1.0',
```

Comment:
Unnecessary `f` in front of the string (might have been to put the version as a variable that would have been defined above?)